job/server/src/test/java/alluxio/master/job/JobMasterTest.java

```diff
@@ -216,7 +198,16 @@ public void submitAndList() throws Exception {
     final List<Long> list = mJobMaster.listCmds(ListAllPOptions.getDefaultInstance());
     Assert.assertEquals(jobIdList, list);
     Assert.assertEquals(TEST_JOB_MASTER_JOB_CAPACITY,
-            mJobMaster.listCmds(ListAllPOptions.getDefaultInstance()).size());
+        mJobMaster.listCmds(ListAllPOptions.getDefaultInstance()).size());
+  }
+
+  private void mockPlanCoordinator() {
```

Comment:
You can make this method return the mocked static, and use the return value in a try-with-resources construct in each test. This eliminates the need to use @Before and @After methods.

Reply:
Good idea, done